src/include/daos/pool_map.h

```diff
@@ -59,6 +59,7 @@ enum pool_component_flags {
 	PO_COMPF_DOWN2OUT	= 1,
 };
 
+#define co_in_ver	co_out_ver
```

Comment:
It seems ok to overwrite this co_out_ver according to current RF implementation, but not quite sure if the co_out_ver needs be preserved when we take new hardware reintegration into account (current RF doesn't handle new hardware reintegration).

I'm fine with current approach since it can fix a defect and doesn't require on-disk format chagnes.